### ISSUE TITLE
Bring in react-testing-library

### ISSUE THREAD

**Issue by MTrost** [200ok-ch/organice#43]
[There's](https://www.youtube.com/watch?v=Agpx6MPNU2s) a new sheriff in UI testing town and it's called [react testing library](https://testing-library.com/docs/react-testing-library/intro).

It's easier and less brittle than react test renderer and enzyme and recommended [by the React team.](https://reactjs.org/docs/testing.html#tools).

As a first step we could

- [x] rewrite a currenty `.skip`ped test with the react-testing-library

...as an example to follow for (re)writing more tests.

**Comment by MTrost** [comment]
I'm sure this would also help with #10

**Comment by munen** [comment]
@MTrost That’s a great idea - and thank you for the input!🙏🏻

I’m very happy to hear that there’s a new way to write better tests. That’s what this project needs foremost before we can start with some more serious refactorings. 

Do you have the time/motivation to get this started?

**Comment by MTrost** [comment]
@munen Yes I do and should be able to open a pull request this week.

**Comment by munen** [comment]
@MTrost This is great news, thank you very much :pray: :bowing_man:

**Comment by munen** [comment]
@MTrost Thank you for your great contribution, Mario! :pray: :bowing_man: 

I do like the approach a lot - also because the react-testing-library approach is very much akin to what we do in other languages/frameworks, already.

I'm very thankful for you nudge in this direction and could take your PR verbatim. I only made one change. The test where you didn't know what it was about, I knew what it was about, so I rewrote it to make sense and showcase what it's about.

I made a gif to showcase the functionality, too. It's basically cycling through [org-todo-keywords](https://orgmode.org/manual/Workflow-states.html).

![demo-cycling-org-todo-sequence](https://user-images.githubusercontent.com/505721/65372529-e604cc00-dc71-11e9-8ab0-d91ed6b5f761.gif)

**Comment by munen** [comment]
@MTrost I would have made the changes to the aforementioned test in a PR so that we could have discussed them before I merged. However, since you didn't have commit rights, you rightfully made a PR from your own repository on which I myself don't have commit rights. Therefore, I thought it would be best to fix the issue and to merge to `master` and then to optimize the workflow.

I just gave you contributor rights to this repository. This means that if you want to do another contribution, you can make it directly in a PR from https://github.com/200ok-ch/organice/ on which we can do a proper review and discuss changes before merging. I added the requirement that every contributor has to make a PR, that it needs a review and that the tests need to be green before merging. With these settings, we have had great success in other projects.

Your own PR would have qualified for all the necessary criteria, of course, because it was well rounded^^

If you're happy with my change to the test, feel free to close this issue. Otherwise, let's make a PR and discuss how to make it better! 

Thanks again and maybe see you in another PR(;

**Comment by MTrost** [comment]
I am very happy with your changes and will close this issue.
